integration-tests/views/common.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  CRONTAB_APIGROUP,
+  CRONTAB_APIVERSION,
+  CRONTAB_KIND,
+  PLUGIN_NAME,
+} from "../../src/const";
+
+export const getNamespacedListPageURL = (testName: string) =>
+  `/k8s/ns/${testName}/${CRONTAB_APIGROUP}~${CRONTAB_APIVERSION}~${CRONTAB_KIND}`;
+
+export const installHelmChart = (path: string) => {
+  cy.exec(
+    `cd ../../crontab-plugin && ${path} upgrade -i ${PLUGIN_NAME} charts/crontab-plugin -n ${PLUGIN_NAME} --create-namespace --set plugin.image=Cypress.env("PLUGIN_TEMPLATE_PULL_SPEC")`,
```

Comment:
We should correspond with the envvar name in https://github.com/openshift/release/pull/42833/files
```suggestion
    `cd ../../crontab-plugin && ${path} upgrade -i ${PLUGIN_NAME} charts/crontab-plugin -n ${PLUGIN_NAME} --create-namespace --set plugin.image=Cypress.env("CRONTAB_PLUGIN_PULL_SPEC")`,
```

Reply:
Good catch.  Updated.